llvm/test/Transforms/ConstraintElimination/transfer-samesign-facts.ll

```diff
@@ -303,3 +303,43 @@ define i1 @ugt_assumed_positive_values(i8 %a, i8 %b) {
 
   ret i1 %result
 }
+
+define i1 @samesign_flipped_signedness_1(i8 %a, i8 %b)  {
```

Comment:
I would call this implied_condition_gt_slt (or whatever are the two icmps). Same for the other test. The current name doesn't covey that much information.